lib/declare_schema/model.rb

```diff
@@ -99,8 +101,29 @@ def index_definitions_with_primary_key
         end
       end
 
-      def primary_key
-        super || 'id'
+      if ::Rails::VERSION::MAJOR < 5
+        def primary_key
+          super || 'id'
```

Comment:
Does this trip up if the model is using a composite primary key? To my understanding, ActiveRecord will warn about the composite key being used and return `nil`. I don't think we want to default to `id` in that case, but respect the composite primary key that was defined.

Reply:
The issue is that the `primary_key` method is built into Rails. And Rails checks for it being `nil`...sometimes raising an exception. I don't think it's acceptable for `declare_schema` to mess with Rails like that ... except this grandfather code that we needed to get Rails 4 to work for the next month or so.

Reply:
Gotcha! Sounds good